easybuild/easyconfigs/f/fastml/fastml-2.3-GCC-11.3.0.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'MakeCp'
+
+name = 'fastml'
+version = '2.3'
+
+homepage = 'http://fastml.tau.ac.il/'
+description = """Maximum likelihood reconstruction of ancestral amino-acid sequences.
+A branch-and-bound algorithm for the inference of ancestral amino-acid
+sequences when the replacement rate varies among sites."""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
+toolchainopts = {'cstd': 'c++03'}
+
+github_account = 'sanger-pathogens'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['c02901d83ab9457852c67e8e14f41a02d2a0653130ed941b288108ba5272611d']
+
+parallel = 1
+
+prebuildopts = "sed -i 's/-depth-32/-depth-64/g' {programs/Makefile*,libs/phylogeny/Makefile} && "
```

Comment:
@lexming could you add an explanatory comment about this please?

Reply:
sure, done in https://github.com/easybuilders/easybuild-easyconfigs/pull/19357/commits/25bdcf006df62129221b56f01cd2a7a2569124f1